src/hooks/use-archive-site.ts

```diff
@@ -48,6 +48,16 @@ export function useArchiveSite() {
 								...snapshot,
 								isLoading: false,
 							} );
+
+							await getIpcApi().showNotification( {
+								title: __( 'Preview created' ),
```

Comment:
The title could also potentially be the name of the local site from which the notification is coming instead of `Preview created`

Reply:
I like it without including the name, but maybe we could consider adding the `site` word in there as well:

```suggestion
								title: __( 'Preview site created' ),
```

Reply:
I passed the site name instead in the title: https://github.com/Automattic/studio/pull/952/commits/8228e476ab8d4d19485b36a9c516c7434ad2f2ae to not repeat "Preview site" in the title and in the notificaiton body.